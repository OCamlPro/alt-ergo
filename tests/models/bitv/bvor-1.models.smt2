(set-logic ALL)
(set-option :produce-models true)
(declare-const x (_ BitVec 8))
(declare-const y (_ BitVec 8))
; This forces the values of x and y to all ones
(assert (= x (bvor #b10100101 y)))
(assert (= y (bvor #b01011010 x)))
(check-sat)
(get-model)
