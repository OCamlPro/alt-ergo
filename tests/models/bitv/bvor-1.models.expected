
unknown
(
  (define-fun x () (_ BitVec 8) #xff)
  (define-fun y () (_ BitVec 8) #xff)
)
